.eslintrc.yml

```diff
@@ -92,7 +92,7 @@ rules:
   vue/require-default-prop: off
   vue/no-v-html: off
   vue/no-use-v-if-with-v-for: off
-  vue/html-closing-bracket-newline: [error, {singleline: never, multiline: never}]
+  vue/html-closing-bracket-newline: error
```

Comment:
Without this, a violation is reported as a warning, which wouldn't fail our CI. The issue is other non-essential rules from this plugin are reported as warnings.

What about removing this instead, and passing a `--max-warnings=0` flag to ESLint calls, to make sure we never get bit by this?
I'm ready to bet we explicitly enabled some other rules in this file (despite using the recommended preset) for this reason too, so we could probably clean it up a little.